Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -572,7 +572,7 @@ class RenderStyle final : public CanMakeCheckedPtr<RenderStyle, WTF::DefaultedOp
 
     // attribute getter methods
 
-    constexpr DisplayType display() const { return static_cast<DisplayType>(m_nonInheritedFlags.effectiveDisplay); }
+    inline DisplayType display() const;
     constexpr WritingMode writingMode() const { return m_inheritedFlags.writingMode; }
     bool isLeftToRightDirection() const { return writingMode().isBidiLTR(); } // deprecated, because of confusion between physical inline directions and bidi / line-relative directions
```

Comment:
Why not also move functions like these into the `Inlines.h` file? How are you drawing the line? (Maybe this should be obvious if I read your change log comment carefully enough.)